registry.json

```diff
@@ -3600,5 +3600,19 @@
     "telegram": "SmaugPool",
     "operatorName": "",
     "github": "SmaugPool"
+  },
+  {
+    "ticker": "MONA",
+    "poolId": "6a84654c217f1923b086e076192b50042911c7b7af3987a1992b77e9",
+    "social": {
+      "website": "https://monada.digital",
+      "twitter": "monadapool",
+      "facebook": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "MonadA",
```

Comment:
Please can you add a user TG handle? This must be the handle of a user and not a group or channel.

Reply:
Now should be okay, thank you